lxc/completion.go

```diff
@@ -241,6 +252,45 @@ func (g *cmdGlobal) cmpInstanceAllKeys(instanceName string) ([]string, cobra.She
 		keys = append(keys, k)
 	}
 
+	return keys, cmpDirectives
+}
+
+func (g *cmdGlobal) cmpServerAllKeys(instanceName string) ([]string, cobra.ShellCompDirective) {
+	var keys []string
+
+	resources, err := g.ParseServers(instanceName)
+	if err != nil || len(resources) == 0 {
+		return nil, cobra.ShellCompDirectiveError
+	}
+
+	resource := resources[0]
+	client := resource.server
+
+	metadataConfiguration, err := client.GetMetadataConfiguration()
+	if err != nil {
+		return nil, cobra.ShellCompDirectiveError
+	}
+
+	server, ok := metadataConfiguration.Configs["server"]
+	if !ok {
+		return nil, cobra.ShellCompDirectiveError
+	}
+
+	keyCount := 0
+	for _, field := range server {
+		keyCount += len(field.Keys)
+	}
+
+	keys = make([]string, 0, keyCount)
```

Comment:
```suggestion
	keys := make([]string, 0, keyCount)
```

Maybe and remove the earlier declaration?

Reply:
I also prefer short variable declarations in functions, thanks for pointing this out!